lib/arclight/traject/ead_formatting.xsl

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" ?>
+<xsl:stylesheet version="1.0"
```

Comment:
Is this an XSLT of our own creation, or some standard one?

Reply:
@cbeer Good question; this is one I created. I don't believe there's any _standard_ EAD->HTML XSLT, though I have at least been able to incorporate some transformation ideas from our (Duke's) legacy finding aids applications, and from [ArchivesSpace source code](https://github.com/archivesspace/archivesspace/blob/master/stylesheets/as-ead-html.xsl).